docs/src/tut/wfs/index.md

```diff
@@ -0,0 +1,879 @@
+# {#wfs} Wisdom For Sale
+
+This tutorial shows you how to build Wisdom for Sale, a command-line and Web-based application that enables two participants (a seller and a buyer) to trade wisdom for tokens via a smart contract running on a consensus network, using Reach.
+Reach programs are built using a Docker container on your computer.
+If you need help installing Reach and its prerequisites then get started at our [Quick Installation Guide](#quickstart).
+
+Wisdom For Sale creates and funds two accounts (one for each participant), enables the seller and buyer to make a transaction, and then exits.
+
+# {#wfs-1} Learning Objectives
+
+The following learning objectives describe what you will be able to do as a result of completing this tutorial.
+
+1. Define essential Reach terms.
+1. Build a simple Reach command-line DApp.
+1. Convert the command-line DApp into a vanilla Web-app.
+
+As you build the DApp, here are some important concepts to keep in mind.
+
+## {#wfs-2} Reach Mode Diagram
```

Comment:
As part of the second draft, be observant of places that intuitive explanations can be inserted. Identify the topics you're teaching directly and indirectly and use those introductory moments as a way to enhance the value of the tutorial for the student. 

IMO, this guide teaches the modes of Reach
The standard library
devnet vs testnet vs mainnet
standard and atomic units
connection modes and networks
How Reach uses Docker
Attaching to a contract
payments and transfers